Created -v option for interactive volume control

jordanhillis opened the pull request.

jordanhillis committed:
Created -v option for interactive volume control

nawar reviewed (commented):
I'm not sure why there's a need for this since pressing the 'u' and 'd' keys should be enough. No?

jordanhillis commented:
I added this as an interactive way to change the volume, this does not replace the -u and -d arguments but instead gives the user a way to change the volume with the up and down arrow keys. You don't have to push it to the master but it's something I added for myself being that it's smoother to change the volume with the arrow keys instead of constantly entering -u and -d arguments each change.
